### ISSUE TITLE
FLAKE: ChannelzChannelTestSweep/ChannelzChannelTest.LastCallStartedMillis

### ISSUE THREAD

**Issue by gnossen** [grpc/grpc#22185]
```
[ RUN      ] ChannelzChannelTestSweep/ChannelzChannelTest.LastCallStartedMillis/1
test/core/channel/channelz_test.cc:313: Failure
Expected equality of these values:
  millis1
    Which is: 1738
  millis2
    Which is: 1737
[  FAILED  ] ChannelzChannelTestSweep/ChannelzChannelTest.LastCallStartedMillis/1, where GetParam() = 8 (7 ms)
```

https://source.cloud.google.com/results/invocations/affc8f3e-9b74-42e0-9e0a-86a06eb70de0/log

**Comment by gnossen** [comment]
Also seen on `macos_opt_native`

**Comment by markdroth** [comment]
Seems like this test is being flaky across a number of platforms and configurations.

**Comment by yashykt** [comment]
should be fixed by #22655
